pyt/lattice.py

```diff
@@ -6,35 +6,34 @@ def __init__(self, cfg_nodes, analysis_type):
         self.el2bv = dict()  # Element to bitvector dictionary
         self.bv2el = list()  # Bitvector to element list
         for i, e in enumerate(analysis_type.get_lattice_elements(cfg_nodes)):
+            # Give each element a unique shift of 1
             self.el2bv[e] = 0b1 << i
-            self.bv2el.append(e)
-        self.bv2el = list(reversed(self.bv2el))
+            self.bv2el.insert(0, e)
```

Comment:
I like that we have codeclimate now, it's so pretty https://codeclimate.com/github/python-security/pyt/coverage/59215a31fd706d000101e863